martian/src/metadata.rs

```diff
@@ -260,13 +273,9 @@ impl Metadata {
         Error::new(e).context(context)
     }
 
-    pub(crate) fn alarm_file(&self) -> &SharedFile {
-        &self.alarm_file
-    }
-
     /// Write a message to the stage alarms.
-    pub fn alarm(&self, message: &str) -> Result<()> {
-        self.alarm_file.appendln(message, true)
+    pub fn alarm(&mut self, message: &str) -> Result<()> {
+        self.append_metadata("alarm", &format!("{} {}", make_timestamp_now(), message))
```

Comment:
Hmm, this was not previously including a timestamp.  I don't think including a timestamp here is a good idea.  Timestamps may be helpful for debugging but they're not especially helpful for user-facing messages, which is what these are.  Most of the time these alarms are indicating something not-quite-right about the inputs; the timestamp is irrelevant to the user.  It's also going to be confusing when mrp prepends the log line noting the alarm with the timestamp of when it's displaying the alarm.